extensions/extensions-docker/pom.xml

```diff
@@ -124,7 +86,8 @@
                             <phase>prepare-package</phase>
                             <configuration>
                                 <dockerfile>${project.basedir}/src/main/resources/docker/Dockerfile</dockerfile>
-                                <dockerBase>python:3.11-slim</dockerBase>
+                                <dockerBuilderBase>python:3.11</dockerBuilderBase>
+                                <dockerFinalBase>python:3.11-slim</dockerFinalBase>
```

Comment:
I: Changes from the new Habushu release allowing us to use the full `3.11` image to build the venv (by having things like `gcc`) while moving the venv onto a slimmer image for the final output.